### ISSUE TITLE
ctc inf/nan：ctc_loss_calculator:144 No valid path found.

### ISSUE THREAD

**Issue by Seventhen** [aaiijmrtt/DEEPSPEECH#1]
I use ds2.py to training a small datasets with 100 of librispeech for only test on cpu , and I reduced rnn layer size to 1. In the training process, I always get ctc loss inf.  
I have also reduced learning rate and change optimizer to MomentumOptimizer with momentum 0.99, but for ctc loss I still got 'inf'
I can't figure it out, can anyone offer some suggestions?
